{% extends "layout.html"%}

{% block content %}

<strong>Usage: seq-gen </strong><br>
[-m MODEL] [-l #] [-n #] [-p #] [-s # | -d #] [-k #] <br>
               [-c #1 #2 #3 | -a # [-g #]] [-i #] [-f e | #] [-t # | -r #] <br>
               [-z #] [-o[p][r][n]] [-w[a][r]] [-x NAME] [-q] [-h] [treefile] <br>
  -l: # = sequence length [default = 1000]. <br>
  -n: # = simulated datasets per tree [default = 1]. <br>
  -p: # = number of partitions (and trees) per sequence [default = 1]. <br>
  -s: # = branch length scaling factor [default = 1.0]. <br>
  -d: # = total tree scale [default = use branch lengths]. <br>
  -k: # = use sequence k as ancestral (needs alignment) [default = random]. <br>
<br>
 <strong>Substitution model options:</strong> <br>
  -m: MODEL = HKY, F84, GTR, JTT, WAG, PAM, BLOSUM, MTREV, CPREV45, MTART, LG, GENERAL <br>
      HKY, F84 & GTR are for nucleotides the rest are for amino acids <br>
  -a: # = shape (alpha) for gamma rate heterogeneity [default = none]. <br>
  -g: # = number of gamma rate categories [default = continuous].<br>
  -i: # = proportion of invariable sites [default = 0.0].<br>
<br>
 <strong>Nucleotide model specific options:</strong><br>
  -c: #1 #2 #3 = rates for codon position heterogeneity [default = none].<br>
  -t: # = transition-transversion ratio [default = equal rate].<br>
  -r: #1 #2 #3 #4 #5 #6= general rate matrix [default = all 1.0].<br>
  -f: #A #C #G #T = nucleotide frequencies [default = all equal].<br>
<br>
 <strong>Amino Acid model specific options:</strong><br>
      specify using the order ARNDCQEGHILKMFPSTWYV<br>
  -r: #1 .. #190 = general rate matrix [default = all 1.0].<br>
  -f: #1 .. #20 = amino acid frequencies e=equal [default = matrix freqs].<br>
<br>
 <strong>Miscellaneous options:</strong><br>
  -z: # = seed for random number generator [default = system generated].<br>
  -o: Output file format [default = PHYLIP]<br>
    p PHYLIP format<br>
    r relaxed PHYLIP format<br>
    n NEXUS format<br>
    f FASTA format<br>
  -w: Write additional information [default = none]<br>
    a Write ancestral sequences for each node<br>
    r Write rate for each site<br>
  -x: NAME = a text file to insert after every dataset [default = none].<br>
  -h: Give this help message<br>
  -q: Quiet<br>
  treefile: name of tree file [default = trees on stdin]<br>

<br>
{% endblock %}

